docs/buildah.md

```diff
@@ -69,6 +69,8 @@ Storage driver option, Default storage driver options are configured in /etc/con
 
 **--userns-uid-map** *mapping*
 
+**NOTE:** This option is not usable when running in rootless mode.  This is only usable when running as root.
```

Comment:
it can be used as rootless

Reply:
The difference with root (as well for `--userns-gid-map`) is that the specified mappings are relative to the rootless user namespace instead of being relative to the host

Reply:
How about:
```
**NOTE:** When this option is specified by a rootless user, the specified mappings are relative to the rootless usernamespace in the container, rather than being relative to the host as it would be when run by a rootful user.
```

@rhatdan @giuseppe  and/or @vrothberg ?

Reply:
rootfull. And then LGTM